src/auth/auth.ts

```diff
@@ -867,3 +869,10 @@ export class Auth extends BaseAuth<AuthRequestHandler> implements FirebaseServic
     return this.tenantManager_;
   }
 }
+
+export function auth(app?: FirebaseApp): Auth {
+  if (app === undefined) {
+    return new Auth(admin.app());
+  }
+  return new Auth(app);
+}
```

Comment:
This would be the top-level function imported by users to instantiate an `Auth` object.